src/apps/FEMSWEEP.cpp

```diff
@@ -0,0 +1,145 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "FEMSWEEP.hpp"
+#include "FEMSWEEP_DATA.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include "common/DataUtils.hpp"
+
+#include <algorithm>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+
+FEMSWEEP::FEMSWEEP(const RunParams& params)
+  : KernelBase(rajaperf::Apps_FEMSWEEP, params)
+{
+  m_ne = params.getFemSweepNumE();
+  m_nd = params.getFemSweepNumD();
+  m_nfds = params.getFemSweepNumFDS();
+  m_na = params.getFemSweepNumA();
+  m_ng = params.getFemSweepNumG();
+
+  setDefaultProblemSize(m_nd * m_ne * m_ng * m_na);
+  setDefaultReps(1);
```

Comment:
TODO: Discuss a reasonable number of repetitions. It is currently set to 1 due to the lengthy runtime of the Sequential variants.